app/src/main/java/org/sopt/and/HomeActivity.kt

```diff
@@ -0,0 +1,190 @@
+) {
+    val context = LocalContext.current
+    val scrollState = rememberScrollState()
+
+    Scaffold(
+        topBar = {
+            Column {
+                CustomTopAppBar(navController = navController)
+                CustomTopAppBarSecond(navController = navController)
+            }
+        },
+        bottomBar = {
+            CustomBottomAppBar(navController = navController)
+        }
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .verticalScroll(scrollState)
+                .background(Color(0xFF1B1B1B))
+                .padding(innerPadding)  // 패딩 적용
+                .padding(all = 10.dp)
+        ) {
+
+            val images = listOf(
+                R.drawable.food_pic1,
+                R.drawable.food_pic2,
+                R.drawable.food_pic3,
+                R.drawable.food_pic4,
+                R.drawable.food_pic5
+            )
+
+            val pagerState = rememberPagerState { images.size }
+
+            HorizontalPager(
+                state = pagerState,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(400.dp)
+            ) { idx ->
+                Image(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .padding(16.dp)
+                        .clip(RoundedCornerShape(16.dp)),
+                    painter = painterResource(id = images[idx]),
+                    contentDescription = "imagePager",
+                    contentScale = ContentScale.Crop
+                )
+            }
+
+            HomeLazyRow(
+                title = "믿고 보는 웨이브 에디터 추천작",
+                images = images,
+                height = 230,
+                width = 140,
+            )
+            Spacer(modifier = Modifier.height(10.dp))
+
+            HomeLazyRow(
+                title = "실시간 인기 콘텐츠",
+                images = images,
+                height = 230,
+                width = 140,
+            )
+            Spacer(modifier = Modifier.height(10.dp))
+
+            HomeLazyRow(
+                title = "오직 웨이브에서",
+                images = images,
+                height = 230,
+                width = 140,
+            )
+            Spacer(modifier = Modifier.height(10.dp))
+
+            HomeLazyRow(
+                title = "오늘의 TOP 20",
+                images = images,
+                height = 260,
+                width = 180,
+            )
+            Spacer(modifier = Modifier.height(10.dp))
+
+            HomeLazyRow(
+                title = "당한 대로 갚아줄게",
+                images = images,
+                height = 230,
+                width = 140,
+            )
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+    }
+}
+
+
+
+@Preview(showBackground = true)
+@Composable
+fun HomeScreenPreview() {
+    val navController = rememberNavController()
```

Comment:
사소하긴 하지만, home의 코드를 복붙한다면 내부 코드가 바뀔 때 마다 해당 작업을 해줘야하고, Preview 코드가 과도하게 길어질 수 있어요.

그렇기 때문에 HomeScreenPreview내부에서 HomeScreen을 선언해주는 방식으로 사용하시면 코드를 줄이고 자동으로 변경점이 반영될 수 있을거에요
```suggestion
@Preview(showBackground = true)
@Composable
fun HomeScreenPreview() {
    val navController = rememberNavController()
    
    HomeScreen(~~)
}
```